frontend/src/components/TransactionLinks/TransactionLink.vue

```diff
@@ -1,19 +1,33 @@
 <template>
   <div class="transaction-link gradido-custom-background">
     <b-row class="mb-2 pt-2 pb-2">
-      <b-col cols="2">
+      <b-col lg="2">
         <type-icon color="text-danger" icon="link45deg" class="pt-4 pl-2" />
       </b-col>
-      <b-col cols="9">
-        <amount-and-name-row :amount="amount" :text="$t('form.amount')" />
-        <memo-row :memo="memo" />
-        <date-row :date="validUntil" :diffNow="true" />
-        <decay-row :decay="decay" />
+      <b-col lg="9" md="9">
+        <b-row>
+          <b-col lg="11" md="10">
+            <amount-and-name-row :amount="amount" :text="$t('form.amount')" />
+            <memo-row :memo="memo" />
+            <date-row :date="validUntil" :diffNow="true" />
+            <decay-row :decay="decay" />
+          </b-col>
+          <b-col lg="1" md="2" class="text-center text-lg-left qr-button">
+            <b-button
+              @click="$bvModal.show('modalPopover-' + id)"
+              class="p-2 test-qr-code"
+              size="sm"
+              variant="outline-success"
+            >
+              <b-img src="img/svg/qr-code.svg" width="60"></b-img>
```

Comment:
Here it would be nice to try a color a bit less black, a bit more grey like `#333333`.

Check this: https://stackoverflow.com/questions/47122852/change-color-of-svg-background-image-bootstrap-4-carousel
Or the more general approach: https://stackoverflow.com/questions/22252472/how-to-change-the-color-of-an-svg-element

Reply:
hinweis: hier nimmt der qr code button eindeutig eine dominante stelle ein.

![FireShot Capture 1104 -  - localhost](https://user-images.githubusercontent.com/1324583/163119628-e4b6c683-ba86-46c5-ac6c-ce076df74c85.png)